pkg/motor/registry_test.go

```diff
@@ -150,19 +150,45 @@ func Test_LoginAndMakeRequest(t *testing.T) {
 		return
 	}
 
-	req := mt.LoginRequest{
-		Did:      ADDR,
-		Password: "password123",
+	req := mt.LoginWithKeysRequest{
+		AccountId: ADDR,
```

Comment:
Shouldn’t this use the DSC not password?

Reply:
It doesn't really make a difference, the login itself isn't being tested here.